lisa/operating_system.py

```diff
@@ -282,15 +282,20 @@ def install_packages(
         package_names = self._get_package_list(packages)
         self._install_packages(package_names, signed)
 
-    def package_exists(
-        self, package: Union[str, Tool, Type[Tool]], signed: bool = True
-    ) -> bool:
+    def package_installed(self, package: Union[str, Tool, Type[Tool]]) -> bool:
```

Comment:
It doesn't need to change the name, if the next one is renamed.

Reply:
package_exists is not clear, so I want to rename it

Reply:
it's a method on os, so it means if the package exists in the current os.

Reply:
revert